### PR TITLE
fix: log error if --config-name is used without multiple configs

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
fix

**Did you add tests for your changes?**
Yes
**If relevant, did you update the documentation?**
NA
**Summary**
 log warning if --config-name is used without multiple configs

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

File: packages/webpack-cli/lib/groups/ConfigGroup.js [hunk 1]
```diff
@@ -163,21 +163,27 @@ const finalize = async (moduleObj, args) => {
...
+            if (namedOptions.length === 0) {
+                logger.error(`Configuration with name "${configName}" was not found.`);
+                process.exit(2);
+            } else {
+                newOptionsObject['options'] = namedOptions;
+            }
         } else {
-            newOptionsObject['options'] = namedOptions;
+            logger.warn('Multiple configurations not found. Please use "--config-name" with multiple configurations.');
+            newOptionsObject['options'] = configOptions;
```
Comment (evilebottnawi) [inline packages/webpack-cli/lib/groups/ConfigGroup.js:179]: Why it is a warning?

Comment (snitin315) [reply packages/webpack-cli/lib/groups/ConfigGroup.js:179]: So that build is not interrupted and stats are printed.

Comment (snitin315) [reply packages/webpack-cli/lib/groups/ConfigGroup.js:179]: Or we can throw error and exit process too.

Comment (evilebottnawi) [reply packages/webpack-cli/lib/groups/ConfigGroup.js:179]: I think it should be error and we should exit, because we can't found config

Comment (anshumanv) [reply packages/webpack-cli/lib/groups/ConfigGroup.js:179]: Yep not exiting will set wrong expectations

Comment (snitin315) [reply packages/webpack-cli/lib/groups/ConfigGroup.js:179]: I will update 👍

Comment (snitin315) [reply packages/webpack-cli/lib/groups/ConfigGroup.js]: updated 👍🏻

**Review by evilebottnawi — APPROVED** [review]
/cc @webpack/cli-team

**Review by anshumanv — APPROVED** [review]

**Pull request merged**